frb_dart/lib/src/loader/_common.dart

```diff
@@ -9,10 +9,14 @@ class ExternalLibraryLoaderConfig {
   /// In Web, the prefix path for the wasm
   final String? webPrefix;
 
-  /// Arguments of `loadExternalLibary`
+  /// The name of the wasm_bindgen module.
+  final String wasmBindgenName;
+
+  /// Arguments of `loadExternalLibrary`
   const ExternalLibraryLoaderConfig({
     required this.stem,
     required this.ioDirectory,
     required this.webPrefix,
+    required this.wasmBindgenName,
```

Comment:
nit: it would be great if we keep the frb_dart/frb_rust runtime lib backward compatible whenever possible. Thus what about making it optional to make old code run, e.g. sth like:

```suggestion
   this.wasmBindgenName = 'wasm_bindgen',
```

Reply:
looking forward to it!

Reply:
You are welcome!